content/docs/command-reference/remote/modify.md

```diff
@@ -589,6 +589,9 @@ more information.
     to the remote in the form: `custom_auth_header: password`.
     `custom_auth_header` and `password` (or `ask_password`) parameters should
     also be configured.
+  - `method` - override
+    [the HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
+    to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.
```

Comment:
Also, might want to mention that PUT could be used with Artifactory. People will find it through search, which should be useful.